soccer/src/soccer/strategy/agent/position/position.cpp

```diff
@@ -71,4 +71,6 @@ bool Position::assert_world_state_valid() {
     return true;
 }
 
+std::string Position::get_name() { return position_name_; }
```

Comment:
```suggestion
std::string Position::get_name() const { return position_name_; }
```
and change the associated .hpp

reason: https://stackoverflow.com/questions/3141087/what-is-meant-with-const-at-end-of-function-declaration